rails-generate: add migration (#3975)
tldr-pages/tldr: pages/common/rails-generate.md

# rails generate

> Generate new Rails templates in an existing project.
> More information: <https://guides.rubyonrails.org/command_line.html#rails-generate>.

- List all available generators:

`rails generate`

> - Generate a new model named Post with attributes title and body:

Reviewer: Maybe we should have gradually increased in complexity by adding another example which adds the attributes. Since there is space for more examples, it could be helpful in easing the users trying to grok the command.

Reviewer 2: Ah, that's a good idea.

I'm not a Ruby programmer, so I didn't pick up on this :P

`rails generate model {{Post}} {{title:string}} {{body:text}}`

- Generate a new controller named Posts with actions index, show, new and create:

`rails generate controller {{Posts}} {{index}} {{show}} {{new}} {{create}}`

> - Generate a new migration that adds a category attribute to an existing Post model:

Reviewer 2: Perhaps this will read better?
```suggestion
- Generate a new migration that adds a category attribute to an existing model called Post:
```

`rails generate migration {{AddCategoryToPost}} {{category:string}}`

> - Generate a scaffold for a model named Post with pre-defined attributes title and body:

Reviewer 2: Perhaps this will read better?
```suggestion
- Generate a scaffold for a model named Post, predefining the attributes title and body:
```
